src/model_selection/mod.rs

```diff
@@ -107,3 +107,43 @@ mod tests {
         assert_eq!(x_test.shape().0, y_test.len());
     }
 }
+
+
+///
+/// KFold Cross-Validation
+///
+/// Entities involved in the KFold procedure:
+///     * a dataset
+///     * a number k of groups (k-folds)
+/// 
+/// Procedure in `cross_validate()`: 
```

Comment:
user's manual: https://github.com/smartcorelib/smartcorelib.org is a better place for the algorithm's pseudocode.